deploy/core-agents-deployment.yaml

```diff
@@ -43,7 +43,7 @@ spec:
             requests:
               cpu: 500m
               memory: 16Mi
-          image: mayadata/mcp-core:release-1.0.2
+          image: mayadata/mcp-core:v1.0.2
```

Comment:
You don't need to modify these yourself, change the release tag on "./scripts/check-deploy-yamls.sh" and then the build should pass.